### PR TITLE
[Maps] Add a banner emphasizing the number of samples that have location data

### PR DESCRIPTION
**Pull request by jshoe**
# Description
- To reduce confusion as to why you might see a small number of samples on the map vs. many samples in the header tab counts, we decided on a banner with the count of samples with location data.
- New font mixins were requested by Jenn in person for the 11px size.

# Notes
Jun-27-2019 14-05-12

# Tests
- Go to data discovery maps, see the banner say the number of samples/projects on the map.

### CODE REVIEW

File: app/assets/src/components/views/discovery/mapping/DiscoveryMap.jsx [hunk 1]
```diff
@@ -142,6 +142,20 @@ class DiscoveryMap extends React.Component {
           </div>
         </div>
       );
+    } else {
+      const idsField = currentTab === "samples" ? "sample_ids" : "project_ids";
+      const count = sumBy(p => size(p[idsField]), values(mapLocationData));
+      return (
+        <div className={cs.bannerContainer}>
```
Comment (tfrcarvalho) [inline app/assets/src/components/views/discovery/mapping/DiscoveryMap.jsx]: Can you make it a component? 
Just think components whenever it is not a custom layout placement :)

**Comment by jshoe** [comment]
@tfrcarvalho moved to a new MapBanner 👍

File: app/assets/src/components/views/discovery/mapping/MapBanner.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import { withAnalytics } from "~/api/analytics";
+
+import cs from "./map_banner.scss";
+
+class MapBanner extends React.Component {
```
Comment (tfrcarvalho) [inline app/assets/src/components/views/discovery/mapping/MapBanner.jsx]: Larger than I thought!

**Comment by jshoe** [comment]
Jenn changed her mind about the info icon so I added it.

Screen Shot 2019-06-28 at 10 47 28 AM

**Pull request merged**